docs/sql-performance-tuning.md

```diff
@@ -40,6 +40,41 @@ Configuration of in-memory caching can be done using the `setConf` method on `Sp
 
 </table>
 
+## Scanning Input Table
+
+Spark SQL can increase the speed while scanning tables via tuning Hadoop configurations.
```

Comment:
I think this only applies when we read Hive table from Spark. BTW, is it something we should document at Spark side if so?

Reply:
Done, I think we can optimize the input format at Spark side (#23506). Maybe we can document these configurations to tell those who want to tune Spark SQL?